### PR TITLE
Update notebook so it uses supplementaries instead of ancillaries

### PR DESCRIPTION
**Pull request by bouweandela**
## Description

Update notebooks/loading-and-processing-data.ipynb so it uses `supplementaries` instead of `ancillaries`.

Fix the issue reported in https://github.com/ESMValGroup/ESMValCore/discussions/1924#discussioncomment-5123552.

Link to documentation: https://esmvaltool--1945.org.readthedocs.build/projects/ESMValCore/en/1945/notebooks/loading-and-processing-data.html

***

## [Before you get started](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#getting-started)

- [x] [☝ Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do

## [Checklist](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#checklist-for-pull-requests)

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [x] [🧪][2] The new functionality is [relevant and scientifically sound](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#scientific-relevance)
- [x] [🛠][1] This pull request has a [descriptive title and labels](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-title-and-label)
- [x] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#code-quality)
- [x] [🧪][2] and [🛠][1] [Documentation](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#documentation) is available
- [x] [🛠][1] Changes are [backward compatible](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#backward-compatibility)
- [x] [🛠][1] The [list of authors](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#list-of-authors) is up to date
- [x] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html#pull-request-checks) were successful

***

To help with the number pull requests:

- 🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValCore/pulls) in this repository

### CODE REVIEW

File: notebooks/loading-and-processing-data.ipynb [hunk 1]
```diff
@@ -615,7 +569,7 @@
...
...ZVmWy8vL5dNOO01OT0+XY2Ji5KFDh8q33nqrXF9f7/Z16uvr5auvvlpOSkqSk5KS5KuvvlpubGzs4e+WelKg59azzz4rFxYWylqtVi4uLpbvvfde2Wg0ul2jvb1dnj9/vpyeni7HxcXJ5513nlxeXu52Hz63BpbuPq/4mkXeeHtOAZBfe+015T42m01+4IEH5NzcXFmn08mnnXaavG3bNrfr9PZrluT4ZoiIiIgGDPYAERER0YDDAIiIiIgGHAZARERENOAwACIiIqIBhwEQERERDTgMgIiIiGjAYQBEREREAw4DICLqdx588EEcd9xxvb0MIopiHIRIRH1KoFOgr732WixevBhGoxEZGRk9tCoi6msYABFRn+J6wvR//vMf3H///di9e7dyW1xcHFJSUnpjaUTUh7AERkR9Sm5urvInJSUFkiR53Na5BHbdddfhwgsvxGOPPYacnBykpqbioYcegsViwd1334309HQUFhbi1VdfdftaFRUVuPzyy5GWloaMjAxccMEFOHToUM9+w0QUEQyAiGhA+Pbbb1FZWYnVq1fj6aefxoMPPojzzjsPaWlp2LhxI373u9/hd7/7HQ4fPgwAaGtrw7Rp05CYmIjVq1dj7dq1SExMxMyZM2EymXr5uyGi7mIAREQDQnp6Ov75z39ixIgRmDdvHkaMGIG2tjbcc889GD58OBYsWICYmBh8//33AIB3330XKpUKr7zyCo499liMGjUKr732GsrLy7Fq1are/WaIqNs0vb0AIqKeMGbMGKhUzt/5cnJyMHbsWOVjtVqNjIwM1NTUAAB++ukn7Nu3D0lJSW7X6ejowP79+3tm0UQUMQyAiGhA0Gq1bh9LkuT1NpvNBgCw2WyYNGkS3nrrLY9rZWVlRW6hRNQjGAAREXkxceJE/Oc//0F2djaSk5N7ezlEFGbsASIi8uLqq69GZmYmLrjgAqxZswYHDx7Ed999h9tuuw1Hjhzp7eURUTcxACIi8iI+Ph6rV69GcXExLr74YowaNQrz5s1De3s7M0JE/QAHIRIREdGAwwwQERERDTgMgIiIiGjAYQBEREREAw4DICIiIhpwGAARERHRgMMAiIiIiAYcBkBEREQ04DAAIiIiogGHARARERENOAyAiIiIaMBhAEREREQDDgMgIiIiGnD+H2uduVoMpa5xAAAAAElFTkSuQmCC\n",
```
Comment (valeriupredoi) [inline notebooks/loading-and-processing-data.ipynb:572]: what in the name of Odin is this massive turd? :rofl:

**Review by valeriupredoi — APPROVED** [review]
cheers @bouweandela = @Peter9192 will you give it a final look and merge, please, mate? :beer:

File: notebooks/loading-and-processing-data.ipynb (hunk 1 above)
Comment (bouweandela) [reply notebooks/loading-and-processing-data.ipynb:572]: an image

Comment (valeriupredoi) [reply notebooks/loading-and-processing-data.ipynb:572]: oh no! I thought it's secret code to communicate with the aliens :alien: But can we not store 12 million lines of `base64` and just produce the image on the fly? At any rate, not about this PR

Comment (bouweandela) [reply notebooks/loading-and-processing-data.ipynb:572]: That would mean downloading the data from ESGF every time we build our documentation, which is a lot more trouble..

**Review by Peter9192 — APPROVED** [review]
Tested, and It works fine. Just a small comment on one more variable name

File: notebooks/loading-and-processing-data.ipynb [hunk 2]
```diff
@@ -237,20 +237,20 @@
...
+    "for anc in tas.supplementaries:\n",
     "    files.extend(anc.files)\n",
     "files = [file for file in files if isinstance(file, ESGFFile)]\n",
     "download(files, CFG['download_dir'])\n",
     "tas.find_files()\n",
     "print(tas.files)\n",
-    "for anc in tas.ancillaries:\n",
+    "for anc in tas.supplementaries:\n",
     "    print(anc.files)"
    ]
```
[the remark this answers is not in the document]
Comment (bouweandela) [reply notebooks/loading-and-processing-data.ipynb]: Well spotted! It should be better now.

**Review by schlunma — APPROVED** [review]
I re-ran all the notebooks, everything works well now :rocket:

**Pull request merged**